### DISCUSSION TITLE
Question about assets and dynamic partitions

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17810] [Q&A]
How many assets are too many assets? Currently I use dynamic partitions but wonder if it would make more sense to just add an additional item to the asset key prefix. Anyone have experienced that?  With dynamic partitions it seems it wouldn’t take a code change but if it was an asset, I would need to either load some factory built assets from some file/configuration somewhere I assume

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699423561956199?thread_ts=1699423561.956199&cid=C01U954MEER).*

**Comment by hellendag** [comment]
Duplicate of https://github.com/dagster-io/dagster/discussions/17809.
